
# Request: Empty (just call to get the current origin)
---
# Response: Current origin of the local coordinate frame (WGS84)
float64 latitude   # Latitude in degrees
float64 longitude  # Longitude in degrees
float64 altitude   # Altitude in meters
bool success       # True if origin is set
string message     # Status or error message
